app/views/chapters/show.html.erb

```diff
@@ -1,5 +1,5 @@
-<h2 class="text-center mt-3"><%= t ".story" %>: <%= @chapter.story.name %></h2>
+<h2 class="text-center mt-3"><%= @chapter.story.name %></h2>
 <h4 class="text-center"><%= @chapter.name %></h4>
 <div class="mt-4" id="chapter-content">
-  <%= @chapter.content %>
+  <%= sanitize @chapter.content %>
```

Comment:
giải thích cho anh cái sanitize này để làm gì =))

Reply:
method **sanitize** dùng để xử lý đầu vào html, bỏ đỏ tất cả các thẻ và thuộc tính không có trong white list, bỏ các thuộc tính href/src với các giao thức không an toàn như javascript: , tất cả các kí tự đặc biệt sẽ được escaped. 
có thể cung cấp các quy tắc tùy chỉnh cho **sanitize**
**sanitize** không đảm bảo rằng đầu vào là hợp lệ và có định dạng tốt.